module-2/backend/week_four.md

```diff
@@ -10,14 +10,26 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 * What is a cookie?
+    * A Cookie is a data structure that is sent back and forth between client and server
```

Comment:
a small piece of data sent from a website and stored on the user's computer by the user's web browser while the user is browsing.